### DISCUSSION TITLE
Version 0.8.15 release tag

### DISCUSSION THREAD

**Discussion by VTimofeenko** [xremap/xremap#432] [Q&A]
Hi!

In the nix flake I track the last version of xremap from the releases. Looks like 0.8.15 was created in 1ea5b1e but there is no corresponding release.

Is something blocking the release?

**Comment by k0kubun** [comment]
It looks like one of the `build` jobs hit a transient failure, and the `publish` job didn't run. I retried failures. It should be uploaded soon. Thanks!

**Comment by VTimofeenko** [comment]
Thank you for looking into this! I'll bump the flake this week.
